.gitignore

```
@@ -129,4 +129,8 @@ dmypy.json
 .pyre/
 
 # Record keeping txt
-/record.txt
\ No newline at end of file
+/record.txt
+
+# Error documents
+err
+errDisc.txt
```

What if we change this to err* in case we want to create more error log files?

---

Sure, we could go a step further include all directories and err*. This kind of specificity could make someone 🤯 when they want to add `errors.py` one day though.
```suggestion
**/err*
```